packages/stash-cli/tsconfig.json

```diff
@@ -8,6 +8,8 @@
     "tsBuildInfoFile": "./build/tsconfig.tsbuildinfo",
 
     // Relaxed rules to allow package to build. FIXME: remove these and fix in code
+    "exactOptionalPropertyTypes": false,
+    "importsNotUsedAsValues": "remove",
```

Comment:
Since these are in each package any reason not to just put them in the root?

Reply:
I figured that if we want to remove this in the future and fix in code, it's much easier to do it package by package basis. I feel like, putting it in the root communicates that we're okay with this relaxed rule.